package.json

```diff
@@ -25,12 +25,11 @@
   },
   "homepage": "https://github.com/fraserxu/react-chartist",
   "peerDependencies": {
-    "react": ">=0.13.0",
+    "react": ">=0.13.0 || ^0.14.0-rc1",
     "chartist": ">=0.9.0"
   },
   "dependencies": {
-    "chartist": "^0.9.4",
-    "react": "^0.13.2"
```

Comment:
It is also reasonable to move this to `devDependencies`, which I am happy to do.